manuscripts2/report.py

```diff
@@ -55,23 +64,23 @@ class Report():
 
     # Helper dict to map a data source class with its Elasticsearch index
     class2index = {
-        git.GitMetrics: GIT_INDEX,
-        github_issues.IssuesMetrics: GITHUB_ISSUES_INDEX,
-        github_prs.PullRequestsMetrics: GITHUB_PRS_INDEX,
+        git: GIT_INDEX,
+        github_issues: GITHUB_ISSUES_INDEX,
+        github_prs: GITHUB_PRS_INDEX
     }
 
     # Helper dict to map a data source name with its python class
     ds2class = {
-        "git": git.GitMetrics,
-        "github_issues": github_issues.IssuesMetrics,
-        "github_prs": github_prs.PullRequestsMetrics,
+        "git": git,
+        "github_issues": github_issues,
+        "github_prs": github_prs
     }
 
     def __init__(self, es_url=None, start=None, end=None, data_dir=None, filters=None,
                  interval="month", offset=None, data_sources=None,
                  report_name=None, projects=False, indices=[], logo=None):
 
-        Query.interval_ = interval
+        self.interval = Query.interval_ = interval
```

Comment:
This does not belong exactly here, but any way: why `Query.interval_` instead of `Query.interval`?

Reply:
??

Reply:
The `interval_` variable of `Query` class can be set for all the child classes, so I wanted it to be different from the `interval` variable of Report class. Hence the underscore at the end.